sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveQuerySuite.scala

```diff
@@ -455,7 +464,7 @@ class HiveQuerySuite extends HiveComparisonTest with BeforeAndAfter {
       |'org.apache.hadoop.hive.serde2.lazy.LazySimpleSerDe' FROM src;
     """.stripMargin.replaceAll("\n", " "))
 
-  test("transform with SerDe2") {
+  ignore("transform with SerDe2") {
```

Comment:
This test is added by me to test working with `org.apache.hadoop.hive.serde2.avro.AvroSerDe` before. However, I found that this serde seems not intended to work with scritp transform. I think we should remove this test.